nb_train.py

```diff
@@ -4,87 +4,159 @@
 import nltk
 import random
 import pickle
-from sklearn.model_selection import train_test_split
 from nltk.corpus import stopwords
 from zipfile import ZipFile
 
-with ZipFile('imdb ratings.zip') as zf:
-    f = zf.open('movie.csv')
-    
-df = pd.read_csv(f)
-smalldf = df[:2000]
-smalldf = smalldf.drop_duplicates(ignore_index = True)
-smalldf.loc[smalldf['label'] == 0, 'label'] = 'neg'
-smalldf.loc[smalldf['label'] == 1, 'label'] = 'pos'
+DATA = 'movie.csv'
+FEATURE_0 = 'text'
+LABEL = 'label'
+LABEL_0 = 0
+LABEL_1 = 1
+NEGATIVE_SENTIMENT = 'neg'
+POSITIVE_SENTIMENT = 'pos'
+DATA_LENGTH = 2000
+STOPWORD_TYPE = 'english'
+TRAIN_DATA_LENGTH = 1500
+NUM_OF_FEATURES = 5
+FILE_NAME = 'nb_classifier'
```

Comment:
`FILE_PATH_TO_STORE`

better yet, can you try making this a command line argument?

https://realpython.com/python-command-line-arguments/